libs/ngxtension/if-validator/src/if-validator.ts

```diff
@@ -1,9 +1,9 @@
 import {
 	AbstractControl,
-	AsyncValidatorFn,
 	FormControl,
-	ValidatorFn,
 	Validators,
+	type AsyncValidatorFn,
+	type ValidatorFn,
```

Comment:
nitpick: this refacto should have been done in a separate PR

Reply:
Yeah, makes sense. I'll turn on `verbatimModuleSyntax` in a different PR